src/loom-provider.ts

```diff
@@ -36,47 +36,108 @@ import { soliditySha3 } from './solidity-helpers'
 import { marshalBigUIntPB } from './big-uint'
 import { SignedEthTxMiddleware } from './middleware'
 
+// Based on https://en.ethereum.wiki/json-rpc#eth-get-transaction-receipt
 export interface IEthReceipt {
   transactionHash: string
   transactionIndex: string
   blockHash: string
   blockNumber: string
-  gasUsed: string
+  from?: string
+  to?: string
   cumulativeGasUsed: string
+  gasUsed: string
   contractAddress: string
   logs: Array<any>
+  logsBloom?: string
+  root?: string
   status: string
 }
 
+// Based on https://en.ethereum.wiki/json-rpc#eth-get-transaction-by-hash
 export interface IEthTransaction {
-  hash: string
-  nonce: string
   blockHash: string
   blockNumber: string
-  transactionIndex: string
   from: string
-  to: string
-  value: string
-  gasPrice: string
   gas: string
+  gasPrice: string
+  hash: string
   input: string
+  nonce: string
+  to: string
+  transactionIndex: string
+  value: string
+  v?: string
+  r?: string
+  s?: string
 }
 
+// Based on https://en.ethereum.wiki/json-rpc#eth-get-block-by-hash
 export interface IEthBlock {
-  blockNumber: string
-  transactionHash: string
+  number: string | null
+  hash: string
```

Comment:
Potentially breaking changes?

Reply:
Those are the values according to the specification, bad I added back the non strict values at the end of the interface

Reply:
Fixed, the potential break can exist only if the client marks as strict like `loomProvider.strict = true`